scripts/dependency_checker.py

```
@@ -301,7 +440,7 @@ def default_run():
 			for wrong_include in wrong_includes:
 				include_prefix, old_include_version, new_include_version, correct_include = find_latest_version(wrong_include, latest_proto_files)
 				if correct_include == False:
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not correct_include:` for false checks

Replace with `not correct_include`

(E712)